quantum/process_keycode/process_tap_dance.c

```diff
@@ -88,6 +88,10 @@ static inline void process_tap_dance_action_on_each_tap(tap_dance_action_t *acti
     _process_tap_dance_action_fn(&action->state, action->user_data, action->fn.on_each_tap);
 }
 
+static inline void process_tap_dance_action_on_each_release(tap_dance_action_t *action) {
+    _process_tap_dance_action_fn(&action->state, action->user_data, action->fn.on_each_release);
```

Comment:
This function checks if `on_each_release` is null btw (see [here](https://github.com/qmk/qmk_firmware/blob/fe98cc5dad538d137ec0484fbda282ef02c1156f/quantum/process_keycode/process_tap_dance.c#L76))